mmdet/models/seg_heads/base_semantic_head.py

```diff
@@ -0,0 +1,83 @@
+from abc import ABCMeta, abstractmethod
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from mmcv.runner import auto_fp16, force_fp32
+
+from ..builder import build_loss
+from ..utils import up_sample_like
+
+
+class BaseSemanticHead(nn.Module, metaclass=ABCMeta):
```

Comment:
Need to deal with the semantic head in roi_heads.

Reply:
We need to be careful to deal with the semantic head in roi_heads to avoid BC-Breaking.